Add nlb@0.1.0

RyanDraves opened the pull request:
Add an initial release of [nlb](https://github.com/RyanDraves/nlb) to the BCR.

@bazel-io skip_check unstable_url

bazel-io commented:
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (nlb) have been updated in this PR.
      Please review the changes. You can view a diff against the previous version in the "Generate module diff" check.

meteorcloudy commented:
Given this is a personal project, do you mind adding your github handle as a prefix to the module name? We generally want to reserve shorter name for well known modules (maybe in future).

RyanDraves commented:
> Given this is a personal project, do you mind adding your github handle as a prefix to the module name? We generally want to reserve shorter name for well known modules (maybe in future).

Sure thing! I wasn't sure of the naming restrictions, so I did a simple lowercase prefix.

RyanDraves commented on modules/ryandraves_nlb/0.1.0/presubmit.yml:
```diff
@@ -0,0 +1,12 @@
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_targets:
+    - '@nlb//...'
```
Thanks. I pushed another repo update after realizing that switching to the BCR `cppzmq` led to undefined `libzmq` references. Do you know where I should submit an issue for this?

meteorcloudy replied:
Looks like it's introduce in https://github.com/bazelbuild/bazel-central-registry/pull/2425, but not by the author of cppzmp.

RyanDraves commented:
I filed #4029, added a test repo, and simplified the test targets for `presubmit.yaml`. It looks like the transitive dependency on `rules_python` requires a test module that sets up that same versions/toolchains and I separately have a bug with some generated source files when compiling my C++ code; I excluded those targets from the test for now.
